.github/workflows/Check.yml

```diff
@@ -0,0 +1,24 @@
+name: Publish a release
+
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
+    tags:
+      - '*'
+
+jobs:
+  macos-build:
+    runs-on: macOS-latest
```

Comment:
I think ubuntu should be fine since there's no iOS component. A while ago we had issues with the availability of macOS boxes, but it might not be an issue anymore.

Reply:
You mean I should do this?
```suggestion
    runs-on: ubuntu
```

Reply:
That would be my preference